graphene/src/box2_d.rs

```diff
@@ -0,0 +1,118 @@
+// Take a look at the license at the top of the repository in the LICENSE file.
+
+use std::fmt;
+
+use glib::translate::*;
+
+use crate::{ffi, Box2D, Point, Rect, Vec2};
+
+impl Box2D {
+    #[doc(alias = "graphene_box2d_get_vertices")]
+    #[doc(alias = "get_vertices")]
+    pub fn vertices(&self) -> &[Vec2; 4] {
+        unsafe {
+            let mut out: [ffi::graphene_vec2_t; 4] = std::mem::zeroed();
+            ffi::graphene_box2d_get_vertices(self.to_glib_none().0, &mut out as *mut _);
+            &*(&out as *const [ffi::graphene_vec2_t; 4] as *const [Vec2; 4])
+        }
+    }
+
+    #[doc(alias = "graphene_box2d_to_float")]
+    pub fn to_float(&self) -> [f32; 4] {
+        unsafe {
+            let mut out = std::mem::MaybeUninit::uninit();
+            ffi::graphene_box2d_to_float(self.to_glib_none().0, out.as_mut_ptr());
+            out.assume_init()
+        }
+    }
+
+    #[doc(alias = "graphene_box2d_init")]
+    pub fn new(min: Option<&Point>, max: Option<&Point>) -> Self {
```

Comment:
I am curious why those would nullable, same for from_vec2

Reply:
Same as in `graphene::Box`. I don't know!

Reply:
```cpp
graphene_box2d_t *
graphene_box2d_init (graphene_box2d_t       *box,
                     const graphene_point_t *min,
                     const graphene_point_t *max)
{
  graphene_vec4_init (&box->minmax,
                      min != NULL ? min->x : 0.f,
                      min != NULL ? min->y : 0.f,
                      max != NULL ? max->x : 0.f,
                      max != NULL ? max->y : 0.f);
  return box;
}
```

Reply:
Not sure what the point is, but that's what it is